pkgs/development/python-modules/matplotlib/default.nix

```diff
@@ -53,14 +53,8 @@
   # Tk
   # Darwin has its own "MacOSX" backend, PyPy has tkagg backend and does not support tkinter
   enableTk ? (!stdenv.hostPlatform.isDarwin && !isPyPy),
```

Comment:
enableTk is left?

Reply:
I can remove it but just wanted to keep the diff minimal. The tkinter dependency is still added optionally.